tasks/main.yml

```diff
@@ -28,8 +34,10 @@
 
 - name: Unmount beegfs data
   mount:
-    path: /data/beegfs/beegfs_data_0
-    state: unmounted
+    path: "{{ item }}"
+    state: absent
+  with_items:
+  - "{{ beegfs_path_oss }}"
```

Comment:
I think it would be better to separate the "uncreate" of a BeeGFS deployment into a separate set of tasks.  If you conditionally include either "create.yml" or "destroy.yml" for example.

Reply:
thats what I was thinking... using e.g. beegfs_state: "present" / "absent" to conditionally invoke "create.yml" and "destroy.yml" playbooks